pkg/etcdcli/etcdcli.go

```diff
@@ -39,6 +39,7 @@ const (
 	BootstrapIPAnnotationKey = "alpha.installer.openshift.io/etcd-bootstrap"
 	DefaultDialTimeout       = 15 * time.Second
 	DefragDialTimeout        = 60 * time.Second
+	DefaultCliTimeout        = 30 * time.Second
```

Comment:
feel free to spell out Client, we're not short on characters today

Reply:
ok， i will use DefaultClientTimeout